.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,22 +1,39 @@
 <!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
 
-## About The Pull Request
+# About the pull request
 
-<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->
+<!-- Remove this text and explain what the purpose of your PR is.
 
-## Why It's Good For The Game
+Mention if you have tested your changes. If you changed a map, make sure you used the mapmerge tool.
+If this is an Issue Correction, you can type "Fixes Issue #169420" to link the PR to the corresponding Issue number #169420.
+
+Prefix the PR title with [admin] if it involves something admin related.
+Prefix the PR title with [s] if you are fixing an exploit, so that it is not announced on the CMSS13 Discord and the server.
```

Comment:
Do we actually have this in our tooling or is this just copypasting?

Reply:
Pretty sure it's just copy pasta.